numpy/lib/type_check.py

```diff
@@ -492,7 +492,8 @@ def _real_if_close_dispatcher(a, tol=None):
 @array_function_dispatch(_real_if_close_dispatcher)
 def real_if_close(a, tol=100):
     """
-    If complex input returns a real array if complex parts are close to zero.
+    If complex input returns a real array if all complex parts of the 
```

Comment:
Might as well take the opportunity to clean up the grammar here a bit as well: consider replacing "If complex input" with "If input is complex,"